GAE/src/org/waterforpeople/mapping/app/web/CronCommanderServlet.java

```diff
@@ -31,18 +31,24 @@
 import org.akvo.flow.dao.ReportDao;
 import org.akvo.flow.domain.persistent.Report;
 import org.waterforpeople.mapping.app.web.dto.SurveyTaskRequest;
-
 import com.gallatinsystems.common.util.S3Util;
+import com.gallatinsystems.device.dao.DeviceDAO;
 import com.gallatinsystems.device.dao.DeviceFileJobQueueDAO;
+import com.gallatinsystems.device.domain.Device;
 import com.gallatinsystems.device.domain.DeviceFileJobQueue;
 import com.gallatinsystems.device.domain.DeviceSurveyJobQueue;
 import com.gallatinsystems.notification.helper.NotificationHelper;
 import com.gallatinsystems.survey.dao.DeviceSurveyJobQueueDAO;
+import com.gallatinsystems.survey.dao.SurveyAssignmentDAO;
 import com.gallatinsystems.survey.dao.SurveyDAO;
 import com.gallatinsystems.survey.dao.SurveyTaskUtil;
 import com.google.appengine.api.datastore.Key;
 
 
+/**
+ * @author stellan
```

Comment:
is this comment necessary? Most codebases don't add the name to the files as git tracks all the changes made to the file. It's just a question of habit, I guess.

Reply:
Ah, didn't notice Eclipse added that